options/locale/locale_en-US.ini

```diff
@@ -652,6 +652,9 @@ issues.label.filter_sort.reverse_alphabetically = Reverse alphabetically
 issues.num_participants = %d Participants
 issues.attachment.open_tab = `Click to see "%s" in a new tab`
 issues.attachment.download = `Click to download "%s"`
+issues.watch = Watch
+issues.watch_issue = Watch issue
+issues.unwatch_issue = Unwatch issue
```

Comment:
subscribe/unsubscribe also matches the icon better (which should otherwise be an eye)